README.adoc

```diff
@@ -1,62 +1,67 @@
+The HiveMQ extension framework provides an open API that allows developers to create custom extensions that suit their specific infrastructures.
+Use our extension framework to extend HiveMQ with custom business logic or to integrate virtually any system into HiveMQ.
 
-With custom HiveMQ extensions, it's easy to add functionality like writing messages to databases, integrate with other service buses, collect statistics, add fine-grained security and virtually anything you else you can imagine.
+The HiveMQ extension framework can be used to add many different types of functionality:
 
-Extension development for HiveMQ is as easy as writing a Java main method once you grasp the core concepts.
-
-=== Using the SDK
-
-An extensive HiveMQ extension documentation is available {hivemq-extensions-docu-link}[on the official HiveMQ website]. It covers the core concepts and shows how to set up your development environment.
-
-You can also clone our example open source HiveMQ extension at {hivemq-github-link}[in our Github page].
-
-
-=== Maven Repository
+* Write messages to a database,
+* integrate other services,
+* collect statistics,
+* add fine-grained security,
+* and much more.
 
-The hivemq-extension-sdk is available in Maven central, so no special configuration is needed.
+Resources:
 
+* Web Site: {hivemq-website}
+* Documentation: {hivemq-extensions-docu}
+* Community Forum: {hivemq-community-forum}
+* Contribution Guidelines: link:CONTRIBUTING.adoc[Contributing.adoc]
+* License: The source files in this repository are made available under the link:LICENSE[Apache License Version 2.0].
 
-=== Using the SDK
+== Using the SDK
 
+The HiveMQ extension SDK is available in Maven central, so no special configuration is needed.
 To use the SDK, add the following dependency to your Maven project:
 
-
 [source,xml]
 ----
-    <dependency>
-        <groupId>com.hivemq</groupId>
-        <artifactId>hivemq-extension-sdk</artifactId>
-        <version>4.3.0</version>
-    </dependency>
-
-
+<dependency>
+    <groupId>com.hivemq</groupId>
+    <artifactId>hivemq-extension-sdk</artifactId>
+    <version>4.3.0</version>
+</dependency>
 ----
 
-=== Extension Development Documentation
+== Extension Development Documentation
+
+Extension development for HiveMQ is as easy as writing a Java main method once you grasp the core concepts.
 
-Please visit {hivemq-extensions-docu-link}[our website] for the documentation how to develop extensions.
+Please visit the extensive {hivemq-extensions-docu}[HiveMQ extension developer documentation].
+It covers the core concepts and shows how to set up your development environment.
 
-=== Example Extensions
+== Example Extensions
 
-Many example extensions are available, most of them as open source. You can find an overview of officially supported extensions {hivemq-extensions-directory}[here]:
+You can find many example extensions by browsing the {hivemq-github}[HiveMQ Github page].
```

Comment:
Is this not the _Github space_?

Reply:
I don't know if the term GitHub space exists, but will change it.